.github/workflows/build-pr.yml

```diff
@@ -22,6 +22,15 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      - name: Get user
```

Comment:
The vulnerability is still present. One can edit the `build-pr.yml` file to change how `user.txt` is fetched.

Reply:
nope it is resolved see how it works, if you observe in deploy-pr.yml you will see in `Get PR Number` part I am using the github api with curl, 

`PR_NUMBER=$(curl -s -H "Authorization: token $GITHUB_TOKEN" \
             "https://api.github.com/repos/${{ github.repository }}/pulls?head=${{env.USER}}:${{ github.event.workflow_run.head_branch }}" \`

in build pr I am fetching the user who raised the pr and stored it in the artifact and upload it
now in deploy-pr  it will download it and now I will put this username and the head_branch of pr which triggered deploy-pr workflow with the help `github.event.workflow_run.head_branch` in the api I mentioned above, hence by this we will get the pr number

Reply:
Ohh okay, Understood. 
So is it safe to merge ? You have tested it properly in your cloned repo ?

Reply:
Okay I have tested so many times